### PR TITLE
#1233 Failing tests fixes

### LINKED ISSUE #1233: Fix failing TestRemoteChartInstall
Noticed that TestRemoteChartInstall is constantly failing in CICD:

```
=== CONT  TestRemoteChartInstall
    install_test.go:78: 
        	Error Trace:	install_test.go:78
        	Error:      	Received unexpected error:
        	            	error while running command: exit status 1; WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /home/circleci/.kube/config
        	            	WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /home/circleci/.kube/config
        	            	Error: INSTALLATION FAILED: failed to download "terratest-0jn8ma/nginx" at version "9.7.4"
        	Test:       	TestRemoteChartInstall
TestRemoteChartInstall 2023-01-31T21:31:31Z logger.go:66: Running command helm with args [delete --namespace testremotechartinstall-vy4hh7 --timeout 5m1s nginx-osz4bd]
TestRemoteChartInstall 2023-01-31T21:31:31Z logger.go:66: WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /home/circleci/.kube/config
TestRemoteChartInstall 2023-01-31T21:31:31Z logger.go:66: WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /home/circleci/.kube/config
TestRemoteChartInstall 2023-01-31T21:31:32Z logger.go:66: Error: uninstall: Release not loaded: nginx-osz4bd: release: not found
    delete.go:11: 
        	Error Trace:	delete.go:11
        	            				panic.go:482
        	            				testing.go:864
        	            				install_test.go:78
        	Error:      	Received unexpected error:
        	            	error while running command: exit status 1; WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /home/circleci/.kube/config
        	            	WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /home/circleci/.kube/config
        	            	Error: uninstall: Release not loaded: nginx-osz4bd: release: not found
        	Test:       	TestRemoteChartInstall
```

### PR DESCRIPTION
**Pull request by denis256**
## Description

Updated helm chart installation test to use actual nginx version.
Updated GCP test to not run instances on regions that don't support `f1-micro` instances.

Fixes #1233.

## TODOs

Read the [Gruntwork contribution guidelines](https://gruntwork.notion.site/Gruntwork-Coding-Methodology-02fdcd6e4b004e818553684760bf691e).

- [x] Update the docs.
- [x] Run the relevant tests successfully, including pre-commit checks.
- [x] Ensure any 3rd party code adheres with our [license policy](https://www.notion.so/gruntwork/Gruntwork-licenses-and-open-source-usage-policy-f7dece1f780341c7b69c1763f22b1378) or delete this line if its not applicable.
- [x] Include release notes. If this PR is backward incompatible, include a migration guide.

## Release Notes (draft)
N/A

### Migration Guide
N/A

### CODE REVIEW

File: test/gcp/packer_gcp_basic_example_test.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ var DefaultRetryablePackerErrors = map[string]string{
 var DefaultTimeBetweenPackerRetries = 15 * time.Second
 
 // Regions that don't support n1-standard-1 instances
-var RegionsToAvoid = []string{"asia-east2", "southamerica-west1", "europe-west8", "europe-southwest1"}
+var RegionsToAvoid = []string{"asia-east2", "southamerica-west1", "europe-west8", "europe-southwest1", "us-east5-a"}
```
Comment (Etiene) [inline test/gcp/packer_gcp_basic_example_test.go:26]: It seems that there are many more regions that don't support n1-standard instances, for example: me-west1, europe-west9, all other us-east5 (not just -a) and many more : https://cloud.google.com/compute/docs/regions-zones

I was wondering if maybe this list should be done differently and include only the regions to test? What is the reason the n1 instance is used here? Can a different instance type be used?

File: modules/gcp/compute_test.go [hunk 2]
```diff
@@ -23,7 +23,7 @@ const DEFAULT_IMAGE_FAMILY_PROJECT_NAME = "ubuntu-os-cloud"
 const DEFAULT_IMAGE_FAMILY_NAME = "family/ubuntu-1804-lts"
 
 // Regions that don't support running f1-micro instances
-var RegionsToAvoid = []string{"asia-east2", "southamerica-west1", "europe-west8", "europe-southwest1"}
+var RegionsToAvoid = []string{"asia-east2", "southamerica-west1", "europe-west8", "europe-southwest1", "us-east5-a"}
```
Comment (Etiene) [inline modules/gcp/compute_test.go:26]: Same as the other comment below

**Comment by denis256** [comment]
GCP tests:

image

Helm tests:

image

**Review by Etiene — APPROVED** [review]
As usual, this will need another approval, but it LGTM now!

**Review by zackproser — APPROVED** [review]
LGTM!

**Pull request merged**